shesha-reactjs/src/designer-components/iconPicker/iconPickerWrapper.tsx

```diff
@@ -26,27 +25,21 @@ interface IconPickerWrapperProps {
     defaultValue?: ShaIconTypes;
 }
 export const IconPickerWrapper: FC<IconPickerWrapperProps> = (props) => {
-    const { customColor, customIcon, fontSize, color, readOnly, applicationContext, value, onChange, borderColor, borderRadius, borderWidth, backgroundColor, stylingBox, defaultValue } = props;
+    const { fontSize, color, readOnly, applicationContext, value, onChange, borderColor, borderRadius, borderWidth, backgroundColor, stylingBox, defaultValue } = props;
     const { data } = useFormData();
     const { globalState } = useGlobalState();
 
     const computedColor = useMemo(() => {
-        return customColor
-        ? executeScriptSync<string>(customColor, applicationContext)
-        : color;
-    }, [applicationContext, customColor, color]);
+        return color;
+    }, [applicationContext, color]);
 
     const computedBorderColor = useMemo(() => {
-        return customColor
-        ? executeScriptSync<string>(customColor, applicationContext)
-        : borderColor;
-    }, [applicationContext, customColor, borderColor]);
+        return borderColor;

```

Comment:
Hi @James-Baloyi if this change is not a mistake - remove unneeded useMemo and use value directly